### PR TITLE
JAMES-2443 Add more configuration elements in DNSService

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by chibenwa** [comment]
Mmmmh

```
no known node for /home/jenkins/build/workspace/1029
```

**mvn tests** are also :green_apple:

**Comment by chibenwa** [comment]
test this please anyway

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

**Comment by aduprat** [comment]
Build triggered. sha1 is merged.

**Comment by aduprat** [comment]
Build started sha1 is merged.

**Comment by aduprat** [comment]
Build finished. No test results found.

File: src/site/xdoc/server/config-dnsservice.xml [hunk 1]
```diff
@@ -61,8 +61,8 @@
            This should always be <strong>false</strong> unless you understand the implications.</dd>
         <dt><strong>maxcachesize</strong></dt>
         <dd>Maximum number of entries to maintain in the DNS cache (typically 50000)</dd>
-        <dt><strong>cacheTTL</strong></dt>
-        <dd>Sets the maximum length of time that records will be stored in the DNS cache in seconds. A negative value disables this feature (that is, sets no limit).</dd>
+        <dt><strong>negativeCacheTTL</strong></dt>
+        <dd>Sets the maximum length of time that negative records will be stored in the DNS negative cache in seconds. A negative value disables this feature (that is, sets no limit).</dd>
```
Comment (rouazana) [inline src/site/xdoc/server/config-dnsservice.xml]: I would say that a negative value activate infinitely this feature, while a zero value deactivate it, no?

**Comment by chibenwa** [comment]
test this please (sorry, killed by mistake)

**Comment by chibenwa** [comment]
test this please

**Comment by aduprat** [comment]
:+1:

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**